storagenode/piecestore/endpoint.go

```diff
@@ -42,11 +42,16 @@ var _ pb.PiecestoreServer = (*Endpoint)(nil)
 type OldConfig struct {
 	Path string `help:"path to store data in" default:"$CONFDIR/storage"`
 
-	WhitelistedSatelliteIDs string        `help:"a comma-separated list of approved satellite node ids" default:""`
-	SatelliteIDRestriction  bool          `help:"if true, only allow data from approved satellites" devDefault:"false" releaseDefault:"true"`
-	AllocatedDiskSpace      memory.Size   `user:"true" help:"total allocated disk space in bytes" default:"1TB"`
-	AllocatedBandwidth      memory.Size   `user:"true" help:"total allocated bandwidth in bytes" default:"500GiB"`
-	KBucketRefreshInterval  time.Duration `help:"how frequently Kademlia bucket should be refreshed with node stats" default:"1h0m0s"`
+	WhitelistedSatelliteIDs string `help:"a comma-separated list of approved satellite node ids" devDefault:"" releaseDefault:"
```

Comment:
Lets remove your SatelliteID @jtolds

Reply:
@stefanbenten can you remove JT satellite from the list? then we can merge it.

Reply:
Yes, let me do that now

Reply:
sorry i'm useless!